pkg/project/create.go

```diff
@@ -191,13 +191,23 @@ func writeCwSettingsIfNotInProject(conID string, projectPath string, BuildType s
 	pathToCwSettings := path.Join(projectPath, ".cw-settings")
 	pathToLegacySettings := path.Join(projectPath, ".mc-settings")
 
+	connection, conErr := connections.GetConnectionByID(conID)
+	if conErr != nil {
+		return &ProjectError{errOpCreateProject, conErr, conErr.Desc}
+	}
+
+	conURL, configErr := config.PFEOriginFromConnection(connection)
+	if configErr != nil {
+		return &ProjectError{errOpCreateProject, conErr, configErr.Desc}
+	}
+
```

Comment:
obv wasn't in this pr but might be better to return a different error code than `errOpCreateProject` as it doesn't say what went wrong.

Also `errOpCreateProject`'s text has a space not an underscore (if you make this change).

Reply:
`errOpConNotFound`

Reply:
Spoke to @jcockbain and we'll keep the operation code the same